### ISSUE TITLE
Commenting does not work

### ISSUE THREAD

**Issue by mlamlamla** [dpinney/gridlabSublime#2]
@dpinney Thanks for providing that profile! I am using Sublime Text 3.1.1, colouring works well, however, (block) commenting does not seem to work, neither through Ctrl+Shft+/ nor through the menu. Thanks for helping!

**Comment by dpinney** [comment]
Yep, you're right, commenting doesn't work. I hope to add this feature. Taking the same code from the C extension would probably work (the GridLAB-D syntax is very similar to C).
